### ISSUE TITLE
[CAIA A11y Research - PILOT]: Debt Resolution, Debt Portal Payment History (July 2024)

### ISSUE THREAD

**Issue by mgleo** [department-of-veterans-affairs/va.gov-team#87817]
> [!IMPORTANT]
> - **TICKET STATUS:** `Confirmed`
> - **PILOT DATE:**  7/16/2024 (Confirmed @EvanAtCoforma) with @rileyorr
> - **TYPES OF ASSISTIVE TECH:** `TBD` - Info on AT being tested in user sessions needed, to prep and conduct pilot `Screen Reader` known prior to.
> - **PROTOTYPE**: [CodePen](https://codepen.io/team/Debt-Team/live/rNgqxLY)

# CAIA A11y Support
This is a child ticket of the research ticket  #88124 - created to track work, completed from this pilot session. (Non-CAIA) Debt resolution ticket: [#87516](https://app.zenhub.com/workspaces/vsa---debt-607736a6c8b7e2001084e3ab/issues/gh/department-of-veterans-affairs/va.gov-team/87516)

> [!TIP]
> **DEFINITION OF DONE**: Refer to the task list in this ticket, for actionable items.

## CAIA Assignee(s)

<details><summary>Toggle to view CAIA info</summary>

@EvanAtCoforma 
@rileyorr 

</details>

> [!CAUTION]
> #### LIMITED CAPACITY JULY 2024 NOTE
> - There will be limited CAIA A11Y capacity the week of 7/15. 
> - There are currently multiple studies the week of 7/15, two studies have scheduled sessions on the CAIA Capacity Calendar and two studies have holds. 
> - Tuesday, July 16; however, we should have time to run a pilot. 
> - @mgleo and @charlottecesana Just let us know as soon as you have your two SR sessions scheduled and we can see if we can cover those as well. Thanks so much! 

## Team Info

<details><summary>Toggle to view team info</summary>

- Team name: Debt Resolution
- OCTO product owner: Denise Coveyduc
- Product name: Debt Portal
- Product manager: Tom Davis
- Moderators: Charlotte Cesana and Megan Gayle
- Note-takers: Megan Gayle, Joseph Lee, Perigean
- Slack channel: [#debt-resolution](https://join.slack.com/share/enQtNzM5MTE3ODIwMDQ4My1hYTY3OGIyZGYzMGYwZjkyYWYyMzg5OTE4MDhiODRiOTU5ZGQ3ZGIzOTMyNTRjMzcwMjA4ZGZkZTE5MjNjY2Fm)
- Dedicated content writer on your team (if you have one): NA
- Dedicated a11y specialist on your team (if you have one): NA

</details>

## Description

<details><summary>Toggle to view the description</summary>

**A CAIA a11y will participate as a user in a pilot of this research study.**

- **Researcher**: The VFS team researcher will test their prototype and conversation guide by conducting the pilot session with a CAIA A11Y.
- **User**: The CAIA A11Y team member will act as a real user in the pilot session.
- **Assistive Tech**: The CAIA A11Y team member will follow the prompts given by the VFS Researcher, while using and demonstrating how the behavior of a `screen reader` would behave.

### Pilot Session
**Date:** Requested 7/16/2024 - `CONFIRMED`
#### Pilot Resources
- Internal CAIA Notes Doc [WIP](https://docs.google.com/document/d/1Y9PYcsDGK95T25ct8dgHiN-wGI0suF-fDMWne2xGKtg/edit)
- Prototype: [CodePen](https://codepen.io/team/Debt-Team/pen/rNgqxLY)
- GitHub Resources
    - [Research Plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/combined_va_debt_portal/payment-history/research/usability-study-6-2024/research-plan.md)
    - [Conversation Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/combined_va_debt_portal/payment-history/research/usability-study-6-2024/at-conversation-guide.md)
    - [Assistive Technology Pilot Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/assistive-tech-pilot-guide.md#assistive-technology-pilot-guide)

</details>

### Assistive Tech Options

> [!TIP]
> **DEVICE SUPPORT DURING RESEARCH SESSIONS**
> - **Available AT**: `Screen readers` on a `desktop` device is currently the primary source of AT that CAIA A11ys are able to support, in addition to `screen magnification` for user research sessions.
> - **Additional Specifications:** If there is a particular `test user`, `browser` or `operating system` you would like us to use, please let us know in advance via this ticket's comments, tagging the assignee - or reach out to @coforma-terry in the CAIA Slack Channel: [#sitewide-content-accessibility-ia ](https://dsva.slack.com/archives/C01K37HRUAH)

## Research Goals

- [ ] How understandable is the:
    - [ ] Overpayment summary page
    - [ ] Overpayment detail page
    - [ ] Resolution page
    - [ ] Debt letter page?
- [ ] Are participants able to locate how to:
    - [ ] Pay their bill (locate the file information that allows them to pay)
    - [ ] Request financial help
    - [ ] Dispute their debt
- [ ] Do participants understand:
    - [ ] When is their bill due?
    - [ ] How much is currently due?

## Definition of Done

```[tasklist]
### A11y Tasks
- [x] **Schedule CAIA a11y:** ~`Tentative` with Evan - calendar hold made and invites sent 7/11/2024. Evan is on PTO, so we will confirm on 7/15 when he is back in the office.~ CONFIRMED 7/15 FOR 7/16/2024 - THANK YOU, EVAN!
- [x] **Share link to [Assistive Tech Pilot Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/assistive-tech-pilot-guide.md) with team, if they haven't reviewed it yet**
- [x] **Determine date and time for pilot:**
- [x] **Update CAIA A11y Capacity Calendar:** Add invite for pilot, when scheduled
- [x] **Review research** [conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/combined_va_debt_portal/payment-history/research/usability-study-6-2024/at-conversation-guide.md) and provide feedback in comments.
- [x] [Request AT device and tech to be used in sessions](LINK)
- [x] [Receive AT device and tech from Team in research ticket](LINK) to be used in sessions to prep for pilot
- [x] Review Goals in Research Plan (Evan and Sara) - added to ticket
- [x] [Plan/create WIP prep doc](https://docs.google.com/document/d/1Y9PYcsDGK95T25ct8dgHiN-wGI0suF-fDMWne2xGKtg/edit) (Evan and Sara)
- [x] **Review prototype** in https://codepen.io/team/Debt-Team/live/rNgqxLY
- [x] **Run pilot** with VFS Researcher
```

**Comment by mgleo** [comment]
Hi @coforma-terry @sara-amanda,
We are hoping to begin the study next week on the 16th. Would someone be available for a Pilot session before then? 
Additionally, can someone join sessions with participants that utilize screen readers?

cc @charlottecesana

**Comment by mgleo** [comment]
Thank you @sara-amanda! I see the hold/invite. 
We will let you know when we have the times for SR sessions.

**Comment by mgleo** [comment]
Hi @sara-amanda 
Here are the times/dates when participants who are using screen readers will be joining:
P3- Wednesday, July 17 3:00 – 4:00pm
P4- Thursday, July 18 1:00 – 2:00pm
P5- Thursday, July 18 4:00 – 5:00pm

I will let you know when I find out what type of screen reader the participants are using.

**Comment by sara-amanda** [comment]
## Pilot Confirmed

@mgleo and @charlottecesana - Evan has confirmed for tomorrow's pilot. I have updated the invite for the pilot. Thank you, @EvanAtCoforma! 

**I see you also sent over the three sessions.** 
- I will update our calendar with holds and check a11y availability, since availability is limited this week, due to PTO. 
- I will update the primary ticket with that information too. #88124

[Per the 7/15/2024 pilot ticket comment ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/87817#issuecomment-2228866446)

> Here are the times/dates when participants who are using screen readers will be joining:
> - P3- Wednesday, July 17 3:00 – 4:00pm
> - P4- Thursday, July 18 1:00 – 2:00pm
> - P5- Thursday, July 18 4:00 – 5:00pm

cc: @NaomiPMC @EvanAtCoforma

**Comment by rileyorr** [comment]
Hi, all! I took some [pilot session notes](https://docs.google.com/document/d/1Y9PYcsDGK95T25ct8dgHiN-wGI0suF-fDMWne2xGKtg/edit#heading=h.dkkcw6h508eu) of accessibility issues and considerations for the team. Some of them are items we chatted about during the call, I just wanted to make sure they were documented for future reference since I know y'all jumped to your first session shortly afterwards. 

I broke the notes up by page we were on. Here's some things of note: 

### Debt overview 

- The focus wasn't visually changing as the participant navigated by headers 
   - @EvanAtCoforma  mentioned he would check his settings to see if that was something he had selected; may also be a Codepen bug
- The alerts are reading out ‘warning’ and ‘success’ - it may be confusing to the participant what ‘success’ means. Does that mean “resolved”? Can a payment be submitted but still have an outstanding balance, so “resolved” may not make sense in that context? This will be interesting to get user feedback on
- One of the questions asked is about how many debts are resolved vs open. The cards need to be navigated through individually, however it’s not readily apparent to the user how many debts exist. Some ideas: 
   - It may be worth exploring displaying the debt cards as a list that would read out the number of debts in the list (ex. 4 items), and perhaps how many are open versus resolved if that information is important to the user. 
   - Evan's feedback: Potentially display the resolved ones first (top of the list) or different sections (open vs resolved) so it’s clear to the Veteran what the overall status is

### Debt details

- Prototype fix: When the user was viewing the ‘Post-9/11 GI Bill debt for housing’ and opened the ‘Why might I have this debt?’ button it was displaying text for an education debt 
- Payments table: The participant navigated right from a list to a table without the table's caption 'Transaction history' being read. This may be a bug 
   - AI: Joseph is going to ping the Platform team to see if it’s a bug they’re currently aware of 
   - Suggestion: Could add a heading 'Transaction history' before the table so it's clearer to the user where they're navigating to
- Question change suggestion: The moderator asked when the next payment is due, which are words the participant was listening for specifically while navigating. After group discussion, it was decided that it may be changed to something like “What is your debt balance and when is it due?” because the phrasing “next payment” isn’t content used directly on the page

### View debt letters 

- The focus isn’t changing when navigating by link - maybe a bug with Codepen?
   - The page scroll was also not working at times 
- When the screen reader read aloud the ‘Person entitled’ data it read ‘Smith’ but the text is HSMITH 
   -  There may be some pronunciation issues for other names in the future. It appears the field is a first initial and last name, so the screen reader may combine the first initial with the last name and read something unrecognizable to the user

Let me or Evan know if you have any questions or want to chat through any of the feedback. Thank you for piloting with us - I look forward to your team's findings once y'all wrap up research! 

cc @sara-amanda @NaomiPMC @mgleo

**Comment by josephrlee** [comment]
@rileyorr, @EvanAtCoforma, thanks for the notes and for being on the call earlier today! I was able to figure out the table issue. It's a bug in the component's v1 version. After updating it to the v3 USWDS version, the table title now gets announced [with the other table details properly](https://codepen.io/team/Debt-Team/live/pomqZqm). 

Here's a screen share of it working:

https://github.com/user-attachments/assets/428f6179-26ee-459d-806b-a1d592fce7d2

**Comment by EvanAtCoforma** [comment]
Awesome! Glad you were able to update the component to the v3 version. If you don't mind, could you let me know what the issue was in the code? Would be nice to know in case I run into any issues again with table titles not being read. A high level description is fine! 

cc: @josephrlee | @rileyorr

**Comment by josephrlee** [comment]
@EvanAtCoforma, not exactly sure, but I think it's just a limitation of the old [v1 table component](https://design.va.gov/storybook/?path=/docs/components-va-table--docs).

**Comment by mgleo** [comment]
Hi @EvanAtCoforma, We had a participant who uses JAWS reschedule to tomorrow, Tuesday at 3pm ET. Would you be able to join?
cc @charlottecesana @josephrlee

**Comment by sara-amanda** [comment]
## Closing Ticket
Thank you for including CAIA A11Ys in your user research. We appreciate your team's dedication, creating with accessibility front-of-mind. We hope to work with you again, soon!

- Closing pilot ticket, now that it has been completed. 
- #88124 
- [Access the research summary comment in the main research ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/88124#issuecomment-2258400764)


cc: @NaomiPMC @rileyorr @EvanAtCoforma @mgleo @charlottecesana @josephrlee
